tests/grate/asm/fs_vs_tests.txt

```diff
@@ -0,0 +1,13 @@
+alu_buffer_size = 1
+pseq_to_dw_exec_nb = 1
+
+.asm
+EXEC
+	MFU:	var unk(0x104E51BA0) r0.fp20, r0.fp20, r0.fp20, r0.fp20
```

Comment:
I *think* I know what most of the unk-stuff here is, but I'll look into that after this PR!